package.json

```diff
@@ -39,6 +39,8 @@
     "truffle-contract": "^4.0.29"
   },
   "dependencies": {
-    "@circles/safe-contracts": "^1.0.4"
+    "@circles/safe-contracts": "^1.0.4",
+    "dotenv": "^8.2.0",
```

Comment:
Think we don't need this to be installed wherever one includes `circles-contracts` as a dependency, would move this to `devDependencies` instead